core/src/main/scala/com/spotify/featran/transformers/NHotEncoder.scala

```diff
@@ -46,6 +46,16 @@ private class NHotEncoder(name: String, encodeMissingValue: Boolean)
 
   import MissingValue.missingValueToken
 
+  def checkForMissingValue(fb: FeatureBuilder[_],
```

Comment:
@slhansen can we change the name of this method since it's actually modifying `FeatureBuilder`? `addMissingValue` perhaps?

Reply:
@nevillelyh it fails locally but won't give me a reason why :(
```
[info] Set current project to featran (in build file:/Users/Sammy/dev_clean/featran/)
[error] /Users/Sammy/dev_clean/featran/core/src/main/scala/com/spotify/featran/transformers/NHotEncoder.scala isn't formatted properly!
[error] (core / Compile / scalafmtCheck) /Users/Sammy/dev_clean/featran/core/src/main/scala/com/spotify/featran/transformers/NHotEncoder.scala isn't formatted properly!```

Reply:
@slhansen you can do `sbt scalafmt` and it will format the code for you.